execution/pegged_orders.go

```diff
@@ -133,7 +130,7 @@ func (p *PeggedOrders) RemoveAllForParty(
 	n := 0
 	for _, o := range p.orders {
 		if o.Party == party /* && o.Status == types.Order_STATUS_PARKED */ {
-			o.UpdatedAt = p.currentTime
+			o.UpdatedAt = p.timeService.GetTimeNow().UnixNano()
```

Comment:
might as well just do this once outside the loops at line 130

Reply:
Fixed here https://github.com/vegaprotocol/vega/pull/5475/commits/e7f35c2b06c503c0e07dc7b3319fe9d237b0f59d
Thanks!